Colors.java

```diff
@@ -0,0 +1,10 @@
+public enum Colors {
```

Comment:
```suggestion
public enum Color {
```
Please, use singular form for enum name